packages/datadog-instrumentations/src/handlebars.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+const shimmer = require('../../datadog-shimmer')
+const { channel, addHook } = require('./helpers/instrument')
+
+const handlebarsReadCh = channel('datadog:handlebars:compile:start')
+
+function wrapCompile (compile) {
+  return function () {
```

Comment:
fyi in a lot of places in the instrumentation the wrapped function is named, just for clarity like so:
```suggestion
  return function wrappedCompile () {
```